test/controllers/notes_controller_test.rb

```diff
@@ -73,4 +73,11 @@ def test_index_paged
     assert_response :success
     assert_select "table.note_list tr", :count => 11
   end
+
+  def test_empty_page
```

Comment:
I created a test method for this scenario. Would your normally split it out like that or bundle this into one of the existing test methods? I'm more accustomed to creating lots of separate `it` blocks in nested `context` blocks in rspec

Reply:
I'm happy with smaller tests. 

I also agree with you with the rspec topic - I use it on all my other projects, and I much prefer having the context blocks with individual tests, rather than ending up with our normal mega-test that are dozens of assertions and 'paragraphs' long. But I'm not sure if there's much appetite for rewriting the test suite yet, until we have a few more developers at least!